README.md

```diff
@@ -66,6 +66,18 @@ Once tasks:
 
 ----
 
+- Run with specific rbac profile and use local rbac mock:
+  ```sh
+  export APP_CLIENTS_RBAC_PROFILE="domain-admin"
+  export APP_ENABLE_RBAC=true  # Or update your secrets/private.mk
+  export RBAC_BASE_URL="http://localhost:8020/api/rbac/v1"
```

Comment:
Several places refer to `localhost:8020`. I guess it's a local instance of the RBAC service. How is a developer suppose to run this service?

Reply:
It is started automatically with the application when `ENV_NAME=local` `APP_ENABLE_RBAC=true`; the value at `RBAC_BASE_URL` is used to extract the listener information.

Reply:
How? I don't see anything in Makefile related to a service.

Reply:
it is started on this line: https://github.com/podengo-project/idmsvc-backend/pull/163/files#diff-674499339ff76ee3d6b200ac84984f5ad68d5a81ad3b696f6aed285e906b53c4R42

Reply:
I think I see what you mean; another approach would be to create a `cmd/mock-rback` and start it from the makefile; I don't know why I didn't do that; that would remove the ENV_NAME check and the code to start the rbac mock in the service. Making the changes.

Reply:
How about a service in a container instead?

Reply:
you wish it, you get it :)

It is now refactored and a container image has been pushed at quay.io/podengo/mock-rback:1.0.0 

See: https://quay.io/repository/podengo/mock-rbac?tab=tags